src/Umbraco.Web.UI.Client/src/assets/lang/en-us.ts

```diff
@@ -1241,6 +1241,12 @@ export default {
 	colorpicker: {
 		noColors: 'You have not configured any approved colors',
 	},
+	colorPickerConfigurations: {
+		colorsTitle: 'Colors',
+		colorsDescription: 'Add, remove or sort colors',
+		showLabelTitle: 'Include labels?',
+		showLabelDescription: 'Stores colors as a Json object containing both the color hex string and label, rather than just the hex string.',
```

Comment:
```suggestion
		showLabelDescription: 'Stores colors as a JSON object containing both the color hex string and label, rather than just the hex string.',
```